backend/src/services/cloudinary/helpers.ts

```diff
@@ -0,0 +1,51 @@
+import StatusCode from '../../utils/StatusCode';
+import fetch from 'node-fetch';
+import { IDetectedImageResponse } from './types';
+import { Request, Response } from 'express';
+import { cloudinary } from '../cloudinary';
+
+const cloudinaryUpload = async (image: string, options?: any) => {
+  return new Promise((resolve: any, reject: any) => {
+    cloudinary.v2.uploader.upload(image, options, (error: any, result: any) => {
+      if (error) return reject(error);
```

Comment:
Using upload_preset im getting an error
{ message: 'Upload preset not found', name: 'Error', http_code: 400 }

What should I do? 
<img width="238" alt="Screenshot 2021-06-05 at 19 25 42" src="https://user-images.githubusercontent.com/19636010/120900217-e6a2e900-c633-11eb-9304-3edd634dc747.png">